easybuild/easyconfigs/g/GBprocesS/GBprocesS-2.3-intel-2020a-Python-3.8.2.eb

```diff
@@ -38,6 +39,20 @@ pretestopts = (
 
 runtest = 'python -m unittest discover -v -s ./test -p test_*.py'
 
+sanity_check_commands = [
+    'export TEMP=$(mktemp -d)'
+    ''' && echo > $TEMP/config.ini <<EOS
```

Comment:
I would just use `%(builddir)s` as temporary directory (since that gets cleaned up automatically).

Also, `EOF` (end-of-file) is more common that `EOS` I think with heredoc